http/mixed-content.json

```diff
@@ -0,0 +1,320 @@
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "auto_upgrade_video_audio": {
+        "__compat": {
+          "description": "Upgrade video and audio content by default.",
+          "spec_url": "https://www.w3.org/TR/mixed-content/#intro",
+          "support": {
+            "chrome": {
+              "version_added": "80"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "127",
+              "notes": [
+                "Set <code>security.mixed_content.upgrade_display_content</code> preference to <code>true</code> to allow HTTP fetching and display of upgradable content.",
+                "Set <code>security.mixed_content.block_display_content</code> preference to <code>true</code> to block all mixed content."
+              ]
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "block_active_content": {
+        "__compat": {
+          "description": "Block active mixed content (later 'blockable' mixed content).",
+          "spec_url": "https://www.w3.org/TR/mixed-content/#intro",
+          "support": {
+            "chrome": {
+              "version_added": "79",
+              "notes": "From version 79 blocks iframes, scripts, and stylesheets."
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "23"
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "block_mixed_downloads": {
+        "__compat": {
```

Comment:
Note, values here are by bisection and using [this tool](https://software.hixie.ch/utilities/js/live-dom-viewer/?%3Ca%20href%3D%22http%3A%2F%2Fexample.com%2F%22%20download%3Etest%3C%2Fa%3E) - Right click on "Test" link and then save -as